src/workerd/util/sqlite.h

```diff
@@ -149,6 +151,12 @@ class SqliteDatabase::Query {
   // Returned pointers (strings and blobs) remain valid only until either (a) nextRow() is called,
   // or (b) a different get method is called on the same column but with a different type.
 
+  ValueOwned getValueOwned(uint column);
```

Comment:
FWIW with a cursor-oriented JS interface that returns one row at a time, you wouldn't need this method that makes an extra copy. You could return StringPtr to JS and it would convert to a V8 string in JSG.